configure.py

```diff
@@ -843,6 +846,7 @@ def _run():
     parser.add_argument("-p", "--venv-path", default=op.join(_ROOTDIR, "venv"), help="Virtual environment directory")
     parser.add_argument("--buildtype", choices=("release", "debug"), default="release", help="Build type")
     parser.add_argument("--coverage", action="store_true", help="Code coverage")
+    parser.add_argument("--sanitize-address", action="store_true", help="Enable address sanitizer")
```

Comment:
can you use the `--sanitize=xxx` form for the argument instead so we can add the `undefined` as well at some point (and eventually threading)

Reply:
Done. The choices are taken directly from the meson documentation:
```
The value of b_sanitize can be one of: none, address, thread, undefined, memory, leak, address,undefined, but note that some compilers might not support all of them. For example Visual Studio only supports the address sanitizer.
```

Reply:
lol weird